.github/workflows/update-browserslist.yml

```diff
@@ -0,0 +1,39 @@
+name: Update Browserslist DB
+on:
+  - workflow_call
+
+jobs:
+  update-dep:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [16]
+        npm: [8.5.x]
+
+    steps:
+    - name: Check out the repo
+      uses: actions/checkout@v2
+      
+    - name: Install node
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node }}
+    
+    - name: Install dependencies
+      run: npm install -g npm@${{ matrix.npm }}
+
+    - name: Install Packages 
+      run: npm ci
+
+    - name: Update dependencies
+      run: npx browserslist@latest --update-db
+
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v4
+      with:
+          token: ${{ secrets.requirements_bot_github_token }}
+          commit-message: 'chore: update browserslist'
+          title: Update browserslist DB
+          body: |
+            Updated browserslist DB
```

Comment:
I would provide a much more verbose body. Is there a SLA to merging that teams should have? What is the importance of the change? Could we do a short summary of the change? (you can echo things to the Actions environment then use them later in the file, for example this line:

https://github.com/openedx/.github/blob/master/.github/workflows/add-depr-ticket-to-depr-board.yml#L69

echos variable $PROJECT_ID to the local $GITHUB_ENV, and then it's used in the next step:

https://github.com/openedx/.github/blob/master/.github/workflows/add-depr-ticket-to-depr-board.yml#L83

Reply:
> Is there a SLA to merging that teams should have?

I believe we could auto-merge the resulting PR from this workflow. The resulting changes from browserslist `--update-db` command update a tiny portion of the package-lock.json file, and is pretty low risk to auto-merge, assuming CI and other status checks pass.

That way, consuming teams won't need to triage/prioritize merging the resulting PR.